README.md

```diff
@@ -1 +1,51 @@
-# My-Portfolio
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My-Portfolio
+
+> This is my first portfolio project in microverse (mobile-first approach).
+  I set up my portfolio repository. My goal here was to set up and customize my portfolio for mobile and desktop versions.
+
+- ![screenshot](/images/project-screenshot.JPG)
+
+## :hammer: Built With
+
+- HTML
+- CSS
+
+## Live Demo 
```

Comment:
- [ ] [REQUIRED] If there is no `Live demo` link for your project, please remove this text(`## Live Demo`) from the `README` file or write down the information that it is not available 😃